recipes-core/rauc/rauc_git.bbappend

```diff
@@ -0,0 +1,18 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+
+SRC_URI:append := "  \
+    file://nvbootctrl-rauc \
+    file://post-install \
+"
+
+RAUC_USE_DEVEL_VERSION = "1"
+
+# nooelint: oelint.append.protvars.SRCREV
+SRCREV = "3ad4979d7cef1b0596707688c6224b1806989df3"
+
+RDEPENDS:${PN} += "tegra-redundant-boot"
+
+do_install:append() {
+    install -m 755 ${WORKDIR}/nvbootctrl-rauc ${D}${bindir}
+    install -m 755 ${WORKDIR}/post-install ${D}${bindir}
+}
```

Comment:
These modifications should be made tegra-specific, i.e., using the `:tegra` override for SRC_URI, RAUC_USE_DEVEL_VERSION, SRCREV, RDEPENDS, and do_install mods.  You should also set `PACKAGE_ARCH:tegra = "${TEGRA_PKGARCH}"` so that the modified version is tegra-specific. Otherwise, other platforms that might be getting built will be affected by this tegra-specific bbapend.

Reply:
Should `setup-nv-boot-control` also be added to the RDEPENDS to pull in the `uefi_common.func` file?

Reply:
Yes, if you change to use the functions there.  We could get fancy and move that file to a separate package to be more granular, but I think we pretty much need what `setup-nv-boot-control` does all the time anyway.